src/components/ChatApp/ChatApp.css

```diff
@@ -151,6 +151,18 @@ header{
     padding: 0;
 }
 
+.message-container.Date{
```

Comment:
the class message-container is containing a false ::after element which is creating a white arrow section. You can avoid using that class.